### DISCUSSION TITLE
Hazine arayan genç sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#127] [Zeka Sorusu]
Maceracı bir genç büyük dedesinden kalma evrakın arasında bir kağıt buldu. Kağıt bir hazinenin yerini tarif ediyordu:
“kuzey … enlemi ve doğu … boylamına doğru yelken aç. orada terk edilmiş bir ada bulacaksın.
adanın kuzey kıyısında yalnız bir meşe ve yalnız bir çam ağacının bulunduğu geniş bir çayırlık alan olacak. yine orada vaktiyle hainleri asmak için kullandığımız bir darağacı göreceksin.
darağacından başla ve adımlarını sayarak meşeye doğru yürü. meşeye ulaşınca tam sağa dön ve aynı sayıda adım at. geldiğin yere bir kazık çak.
tekrar darağacına git ve adımlarını sayarak çama doğru yürü. çama ulaşınca tam sola dön ve aynı sayıda adım at. geldiğin yere bir kazık daha çak.
iki kazığın arasında tam orta noktayı kaz. hazine orada.”
Ada ve ağaçlar oradaydı, ama maalesef darağacı yoktu ve yerini belli edecek hiç bir iz de bulunmuyordu. Çünkü geçen zamanın yağmur ve rüzgarları onu aşındırıp çürütmüş, bütün izlerini silip süpürmüştü. Genç rasgele bir sürü yeri kazdı ama ada büyüktü ve sonuçta pes ederek elleri boş geri döndü.
Keşke, diyorum, delikanlı matematik derslerinden kaytarmasaydı da bu hikaye mutlu sonla bitseydi.
Hikaye, George Gamow’un bir, iki, üç, … sonsuz kitabından alındı (the viking press, 2. baskı, kasım 1962, s. 51).
![s8nmu1748xi9](https://github.com/sonsuzus/sonsuzus/assets/52765796/8e4bdc9e-c0b0-4bf9-b3d4-df6e7db8b702)

**Comment by tasgin123** [comment]
![image](https://github.com/sonsuzus/sonsuzus/assets/145910740/3b49bd2f-5240-40f6-ab5e-2d121052989d)

hazine her türlü aynı yere gelir
